wolfssl/wolfcrypt/md5.h

```diff
@@ -73,9 +73,12 @@ typedef struct Md5 {
     #include "wolfssl/wolfcrypt/port/ti/ti-hash.h"
 #endif
 
-WOLFSSL_API void wc_InitMd5(Md5*);
-WOLFSSL_API void wc_Md5Update(Md5*, const byte*, word32);
-WOLFSSL_API void wc_Md5Final(Md5*, byte*);
+
+WOLFSSL_API int wc_InitMd5(Md5*);
```

Comment:
Changing these APIs to have return codes means that all calls in the library proper (src/*.c ; wolfcrypt/src/*.c) should now be checking the return codes as well.  And all tests as well.  See SHA.  Some constant time functions won't explicitly check return codes but those are noted.